views/index.html

```diff
@@ -235,9 +235,12 @@
 <ol>
   <li>From the Web Console homepage, navigate to your project</li>
   <li>Click on Browse &gt; Builds</li>
-  <li>From the view for your Build click on the button to copy your GitHub webhook</li>
-  <li>Navigate to your repository on GitHub and click on repository settings &gt; webhooks</li>
-  <li>Paste your webhook URL provided by OpenShift &mdash; that's it!</li>
+  <li>Click the link with your BuildConfig name</li>
+  <li>Click the Configuration tab</li>
+  <li>Click the "Copy to clipboard" icon to the right of the "GitHub webhook URL" field</li>
+  <li>Navigate to your repository on GitHub and click on repository settings &gt; webhooks &gt; Add webhook</li>
+  <li>Paste your webhook URL provided by OpenShift</li>
+  <li>Leave the content type as "application/json" and adjust the secret, SSL verifictation, and other Webhook parameters as necessary for your specific environment</li>
```

Comment:
If the defaults work with OpenShift, we might just say that you can take the defaults (unless there is a value they really should be changing).